### PR TITLE
Final Project for Submission. 

### PR DESCRIPTION
**Pull request by JonathanLadwig**
No more changes until after code review, unless you guys find something.

### CODE REVIEW

File: src/app/components/calendar/calendar.component.ts [hunk 1]
```diff
@@ -26,14 +26,13 @@ export class CalendarComponent implements OnInit {
...
   selectChange($event: Date) {
-    console.log($event);
     this.selectedDate = $event;
     this.getActivitiesByDate();
   }
 
   ngOnInit(): void {
     const holidayID = this.holidayService.getSelectedHolidayID();
-    this.store.dispatch(loadActivitiesByHolidayID({ idHoliday: holidayID }));
+    this.store.dispatch(loadActivitiesByHolidayID({ idHoliday: holidayID }));    //use a more correct selector rather?
```
Comment (MicNow27) [inline src/app/components/calendar/calendar.component.ts]: Remember to remove comments before last commit

File: src/app/components/create-new-activity-form/create-new-activity-form.component.ts [hunk 2]
```diff
@@ -89,15 +114,18 @@ export class CreateNewActivityFormComponent {
     this.validateForm.reset();
     this.store.dispatch(addActivity({ newActivity: activity }));
     this.activityService.addActivity(activity);
+    this.successful = true;
+    // this.createSuccessNotification();
```
Comment (MicNow27) [inline src/app/components/create-new-activity-form/create-new-activity-form.component.ts]: Remove, please.

File: src/app/components/holiday-tab/holiday-tab.component.ts [hunk 3]
```diff
@@ -46,6 +45,7 @@ export class HolidayTabComponent implements OnInit {
 
   cancelDelete() {
```
Comment (MicNow27) [inline src/app/components/holiday-tab/holiday-tab.component.ts]: There's no method body

File: src/app/components/leaflet-map/leaflet-map.component.html [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<!-- <div class="map-container h-64 w-full rounded-lg" leaflet (leafletClick)="onMapClick($event)" [leafletOptions]="options"
```
Comment (MicNow27) [inline src/app/components/leaflet-map/leaflet-map.component.html]: Commented out code

File: src/app/components/create-new-activity-form/create-new-activity-form.component.ts [hunk 5]
```diff
@@ -22,13 +25,32 @@ export class CreateNewActivityFormComponent {
...
+    private currencyService: CurrencyService,
+    private notification: NzNotificationService,
+    private message: NzMessageService) {
     this.currencyOptions$.subscribe(data => {
       this.currencies = Object.keys(data.symbols) as string[];
+      this.successful = false;
     })
   }
 
+  beforeConfirm(): Observable<boolean> {
```
Comment (MicNow27) [inline src/app/components/create-new-activity-form/create-new-activity-form.component.ts]: This method isn't used

**Review by MicNow27 — CHANGES_REQUESTED** [review]

**Comment by JonathanLadwig** [comment]
Thank you for the review. Requested changes were implemented with some additional uncaught problems through testing.

**Review by MicNow27 — APPROVED** [review]

**Pull request merged**